src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.ts

```diff
@@ -47,4 +52,21 @@ export class DivikComponent implements OnInit {
       type: 'heatmap'
     }];
   }
+
+  displayConfig(config: DivikConfig): string {
+    return 'Divik config:' +
```

Comment:
you can use template strings : https://basarat.gitbooks.io/typescript/docs/template-strings.html.